src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -69,11 +68,22 @@ export class SearchMenu extends React.Component {
     }
   }
 
-  handleInputChange = e => {
-    this.setState({ userInput: e.target.value });
+  handleInputChange = event => {
+    this.setState({ userInput: event.target.value });
+  };
+
+  formatTypeaheadSuggestion = suggestion => {
+    const { userInput } = this.state;
+    const remainder = suggestion.replace(userInput, '');
+    return (
+      <>
+        <strong>{userInput}</strong>
+        {remainder}
+      </>
+    );
   };
```

Comment:
In testing do you find that the user's query is always part of the suggestion? Just wondering if there are edge cases with suggestions being singular while queries are plural or something.

Reply:
From my testing they were, but there is certainly some quick win improvements that can be made here!